flask-server/introverseSQL.sql

```diff
@@ -61,39 +61,39 @@ VALUES
 ("Just here for cool community!", "Introduce", "DogWar");
 
 -- Content tables for recommendations
-CREATE TABLE Books ( 
-	Book_ID INTEGER PRIMARY KEY NOT NULL,
-	Book_Name VARCHAR(100) UNIQUE NOT NULL,
-    Book_Author VARCHAR(30),
-    Book_Genre VARCHAR(25),
-	Price FLOAT NOT NULL, 
-    Book_Script VARCHAR(1000),
-    Book_Image VARCHAR(100) UNIQUE
+CREATE TABLE books ( 
+	book_id INTEGER PRIMARY KEY NOT NULL,
```

Comment:
I'm not a huge fan of prefixing the attribute names with the entity name. It should be somewhat obvious that this id refers to the book it's part of. 

I would reserve this notation for when creating foreign keys. E.g. if there is a reference to the author, that could be the author_id.

Reply:
Yeah that's fair, makes a lot of sense. I'll move away from doing that in the future and practice with some more SQL projects.